utils.go

```diff
@@ -24,14 +25,27 @@ func reverseAnimeSlice(s []*mal.Anime) {
 	}
 }
 
-func homeDir() string {
+func getDataDir() string {
+	// Check for old cache dir at $HOME/.mal
 	usr, err := user.Current()
 	if err != nil {
-		log.Printf("Error getting current user: %v", err)
-		return ""
+		log.Printf("Error getting current user: %v. ignoring", err)
+	} else {
+		oldDir := filepath.Join(usr.HomeDir, ".mal")
+		_, err := os.Stat(oldDir)
+    if err == nil { return oldDir }
+    if os.IsExist(err) {
+			log.Printf("Error checking for old cache dir: %v, ignoring", err)
+		}
 	}
 
-	return usr.HomeDir
+	// Old dir isn't there, use new $XDG_CACHE_HOME/mal
+	dir, err := os.UserCacheDir()
```

Comment:
I think we should use $XDG_CONFIG_HOME, not $XDG_CACHE_HOME. 
```suggestion
	dir, err := os.UserConfigDir()
```

Reply:
Yeah this seems like a great change, just accepted it